src/extension.ts

```diff
@@ -1,96 +1,104 @@
-import * as vscode from 'vscode';
-import * as fs from 'fs';
-import * as os from 'os';
-import { exec } from 'child_process';
+import * as vscode from "vscode";
+import * as fs from "fs";
+import * as os from "os";
+import { exec } from "child_process";
```

Comment:
I realize that formatting is always a matter of taste; I'm happy to drop this commit if you don't like it.